cloud-core/src/main/java/cloud/commandframework/Command.java

```diff
@@ -318,6 +328,32 @@ public Command(
         return new ArrayList<>(this.arguments);
     }
 
+    /**
+     * Return a copy of the command argument array, ignoring flag arguments.
+     *
+     * @return Copy of the command argument array, ignoring flag arguments. This List is mutable.
```

Comment:
Repeating the main doc in the return is redundant, something simple like `argument list` should suffice.